docs/pages/machine-id/workload-identity.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Workload Identity
+description: Teleport's Workload Identity for SPIFFE
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to [hear your feedback](mailto: product@goteleport.com).
+We currently do not recommend using this feature in production.
+</Admonition>
+
+Teleport's Workload Identity issues flexible short-lived identities intended
+for workloads. It is compatible with the industry-standard SPIFFE specification
+meaning that it can be used in place of other SPIFFE compatible identity
+providers.
+
+Teleport's Workload Identity brings a number of Teleport features you are
+already familiar with, such as:
+
+- Rich auditing of the issuance of SPIFFE SVIDs.
+- Powerful RBAC for limiting access to SPIFFE IDs to specific machines and
+  users.
+- Easy rotation and management of the SPIFFE CA.
+- Using Teleport SPIFFE CA with 3rd party services such as AWS Roles Anywhere.
+
+## Overview
```

Comment:
I think we can remove this H2 and elevate the remaining H3s to H2s. If we want to keep this an H2, I would add an intro paragraph below it.

Reply:
2285838b13